mypy/config_parser.py

```diff
@@ -329,7 +334,7 @@ def get_prefix(file_read: str, name: str) -> str:
     else:
         module_name_str = name
 
-    return f"{file_read}: [{module_name_str}]: "
```

Comment:
What's the purpose of the change to this line?

Reply:
Before it was `print("%sfoobar" % prefix)`, now it's `print(prefix, "foobar")` hence removed the space.